ModuleConfig.cfc

```diff
@@ -1,22 +1,31 @@
 component {
 
-    this.name = "cfmigrations";
-    this.author = "Eric Peterson";
-    this.description = "Keep track and run your database migrations with CFML";
-    this.version = "0.0.0";
-    this.cfmapping = "cfmigrations";
-    this.autoMapModels = false;
-    this.dependencies = [ "qb" ];
-
-    function configure() {
-        settings = {
-            migrationsDirectory = "/resources/database/migrations",
-            defaultGrammar = "BaseGrammar"
-        };
-
-        binder.map( "MigrationService@cfmigrations" )
-            .to( "#moduleMapping#.models.MigrationService" )
-            .initArg( name = "defaultGrammar", ref = "#settings.defaultGrammar#@qb" );
-    }
+	this.name          = "cfmigrations";
+	this.author        = "Eric Peterson";
+	this.description   = "Keep track and run your database migrations with CFML";
+	this.version       = "0.0.0";
+	this.cfmapping     = "cfmigrations";
+	this.autoMapModels = false;
+	this.dependencies  = [ "qb" ];
+
+	function configure() {
+		settings = {
+			defaultGrammar      : "BaseGrammar"
+		};
+
+	}
+
+	function onLoad(){
+
+		if( settings.keyExists( "defaultGrammar" ) && listLen( settings.defaultGrammar, "." ) < 2 ){
```

Comment:
Was this in here before?  The requirement to add `@qb` to grammars was introduced in a prior breaking change.  I don't really know I want to try to pad or fix it here.

Reply:
I moved this to `onLoad` because the `settings` reference in `configure` was using the `settings` struct defined there.  The custom config variables weren't merged yet.

Reply:
This needs to be here and matches the old methodology of appending `@qb`.  I'm marking this one as resolved.